dask_expr/tests/test_collection.py

```diff
@@ -348,6 +379,8 @@ def test_repr(df):
 
 
 def test_combine_first_simplify(pdf):
+    if BACKEND == "cudf":
```

Comment:
Can we do this as a decorator? 

```
@pytest.mark.xfail(BACKEND_IS_CUDF)
```

Reply:
Yeah, can/should definitely do this for the general case, but I *think* we still need to add an `if` statement for a few parameter-specific cases.